samples/Microsoft.AspNet.SignalR.Samples/Scripts/hubs.js

```diff
@@ -1,5 +1,5 @@
 /*!
- * ASP.NET SignalR JavaScript Library v2.2.0-pre
```

Comment:
Does running build.cmd automatically update this? Any idea why this wasn't updated before?